components/log-viewer-webui/package.json

```diff
@@ -6,7 +6,7 @@
     "client:lint:check": "cd client && npm run lint:check",
     "client:lint:fix": "cd client && npm run lint:fix",
     "client:start": "cd client && npm start",
-    "init": "npm i && (cd client && npm i) && (cd server && npm i)",
+    "init": "npm ci && (cd client && npm ci) && (cd server && npm ci)",

```

Comment:
Not mandatory, but to follow our conventions, shall we use the long-form command name (`clean-install`)?